damusTests/EventGroupViewTests.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  EventGroupViewTests.swift
+//  damusTests
+//
+//  Created by Terry Yiu on 2/26/23.
+//
+
+import XCTest
+@testable import damus
+
+final class EventGroupViewTests: XCTestCase {
+
+    override func setUpWithError() throws {
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+    }
+
+    override func tearDownWithError() throws {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+    }
+
+    func testReactingToText() throws {
```

Comment:
This test is not exhaustive yet on all possible combinations. But putting out the PR preemptively in case Will wants to pull it in prior to release.